packages/magika.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,19 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = 'magika'
+    $category = 'File Information'
+    $executablePath = $toolName
+    $arguments = " -h"
+    $iconLocation = Join-Path ${Env:WinDir} "system32\cmd.exe" -Resolve
+
+    # Create output file to log python module installation details
+    $outputFile = VM-New-Install-Log ${Env:VM_COMMON_DIR}
+
+    Invoke-Expression "py -3.10 -m pip install $toolName 2>&1 >> $outputFile"
```

Comment:
What about moving this to a `VM-Pip-Install` helper so that installing python libraries in the future is as easy as `VM-Pip-Install $toolName`